File: doc/codes/csharp/examples/api/command/silencer-example.cs
using System.Numerics;
using System.Threading.Tasks;
using AUTD3;
using AUTD3.Link;

// HIDE
namespace AUTD3.DocSamples.ApiCommandSilencerExample;

internal static class Sample
{
    internal static async Task Run()
    {
        // HIDE_END
var geometry = new Geometry(new[] { new Device(Vector3.Zero) });
var client = await Client.OpenAsync(geometry, NopLink.Create(), new ClientConfig());

var builder = client.DatagramBuilder();
builder.Push(new SetSilencer());
var frames = builder.Build();
foreach (var frame in frames)
{
    await client.SendCheckedAsync(frame);
}

await client.CloseAsync();
        // HIDE
    }
}
// HIDE_END

File: doc/codes/python/examples/api/command/silencer-example.py
import asyncio

from autd3 import Client, ClientConfig
from autd3.commands import SetSilencer
from autd3.geometry import Autd3, Geometry
from autd3_link_nop import Nop


async def main() -> None:
    geometry = Geometry([Autd3([0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0])])
    async with await Client.open(geometry, Nop(), ClientConfig()) as client:
        builder = client.datagram_builder()
        builder.push(SetSilencer())
        frames = builder.build()
        for frame in frames:
            await client.send_checked(frame)


asyncio.run(main())
